contribute.md

```diff
@@ -162,19 +166,39 @@ sudo -u postgres createdb foreman
 sudo -u postgres createdb foreman-test
 ```
 
+#### Libraries
+
+Some Ruby libraries (rubygems) require development tools (compilers, make), libraries and header files and Chrome driver to be present on the host.
+
+Instructions for Enterprise Linux 9:
+
+```
+# Enable CRB (RHEL)
+subscription-manager repos --enable codeready-builder-for-rhel-9-$(arch)-rpms
+# Enable CRB (Almalinux, CentOS Stream, Rocky Linux)
+dnf install dnf-plugins-core
+dnf config-manager --set-enabled crb
+# Install tools
+dnf groupinstall "Development Tools"
+dnf install libvirt-devel postgresql-devel openssl-devel libxml2-devel \
+  sqlite-devel libxslt-devel zlib-devel readline-devel systemd-devel libcurl-devel krb5-devel
+dnf install chromedriver
```

Comment:
This is in EPEL so I'd at least add a note

Reply:
Done